.github/workflows/notes.md

```diff
@@ -6,8 +6,26 @@ ${NVIM_VERSION}
 
 ### Windows
 
-1. Extract **nvim-win64.zip**
-2. Run `nvim-qt.exe`
+#### Zip
+
+1. Download **nvim-win64.zip**
+2. Extract the zip.
+3. Run `nvim-qt.exe`
+
+#### MSI
+
+1. Download **nvim-win64.msi**
+2. Run the MSI
+3. Add the Neovim location to your path.
+   - Default location is `C:\Program Files\Neovim`
+4. Run `nvim` or `nvim.exe`
+
+#### NSIS
+
+1. Download **nvim-win64.exe**
+2. Run the installer.
+   - Ensure that the option to add the installation location to your path is checked if it's your first installation.
+3. Run `nvim` or `nvim.exe`
```

Comment:
Do either the MSI or NSIS installers add entries to the Start Menu?  If so, is it for nvim or nvim-qt (or both)?

Reply:
Yes, you can search for both executables for both the MSI and the NSIS installations. Here's a screenshot of the apps when I search them using Windows 11. I'd _assume_ that a Windows user would search for `nvim-qt.exe` if they were to search using the start menu. @jamessan thinking about the `notes.md` instructions here again, it would make sense to add something along the lines of "Search for nvim-qt.exe using the Windows start menu or run `nvim` from your shell of choice."

What do you think about that change?

![nvim-qt_screenshott](https://user-images.githubusercontent.com/19373652/154913489-19c782e7-cf36-4fb6-8986-127644e81934.png)
![nvim_screenshot](https://user-images.githubusercontent.com/19373652/154913497-2fe31c21-0a57-442e-9454-dc04e0744898.png)